UpdateService/handler.ts

```diff
@@ -148,9 +150,22 @@ const updateServiceTask = (
  * Handles requests for updating a service by given serviceId and a Service Payload.
  */
 export function UpdateServiceHandler(
+  telemetryClient: ReturnType<typeof initAppInsights>,
   apiClient: APIClient
 ): IUpdateServiceHandler {
   return (_, apiAuth, ___, userAttributes, serviceId, servicePayload) => {
+    const trackUpdateServiceVisibility = (
+      isVisible: boolean,
+      svcId: ServiceId
+    ): void =>
+      telemetryClient.trackEvent({
+        name: "api.services.update",
+        properties: {
+          isVisible: isVisible ? "false" : String(isVisible),
```

Comment:
I'm sure there's meaning on that, but I cannot really get it

Reply:
Nope you're right 🤣